openforcefield/topology/topology.py

```diff
@@ -1578,7 +1581,8 @@ def from_mdtraj(mdtraj_topology, unique_molecules=None):
         return Topology.from_openmm(
             mdtraj_topology.to_openmm(), unique_molecules=unique_molecules)
 
-    def to_mdtraj(self):
+    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API. Fix?
```

Comment:
Why?

Reply:
Added:
```
    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API.
    #       Before exposing this, we should look carefully at the information that is preserved/lost during this
    #       conversion, and make it clear what would happen to this information in a round trip. For example,
    #       we should know what would happen to formal and partial bond orders and charges, stereochemistry, and
    #       multi-conformer information. It will be important to document these risks to users, as all of these
    #       factors could lead to unintended behavior during system parameterization.
```

Reply:
Perfect!